### ISSUE TITLE
Error code 1

### ISSUE THREAD

**Issue by LennardWesterveld** [wantedly/step-pretty-slack-notify#24]
Hi there,
Sorry i cannot look into because i don't have time atm, but just let you guys know ;)

```
export WERCKER_STEP_ROOT="/pipeline/pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247"
export WERCKER_STEP_ID="pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247"
export WERCKER_STEP_OWNER="wantedly"
export WERCKER_STEP_NAME="pretty-slack-notify"
export WERCKER_REPORT_NUMBERS_FILE="/report/pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247/numbers.ini"
export WERCKER_REPORT_MESSAGE_FILE="/report/pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247/message.txt"
export WERCKER_REPORT_ARTIFACTS_DIR="/report/pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247/artifacts"
export WERCKER_PRETTY_SLACK_NOTIFY_WEBHOOK_URL="***"
export WERCKER_PRETTY_SLACK_NOTIFY_CHANNEL="***"
export WERCKER_PRETTY_SLACK_NOTIFY_USERNAME="****"
export WERCKER_PRETTY_SLACK_NOTIFY_BRANCHES=""
export WERCKER_PRETTY_SLACK_NOTIFY_NOTIFY_ON="failed"
source "/pipeline/pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247/run.sh" < /dev/null
Ruby Version: ruby 1.9.3p194 (2012-04-20 revision 35410) [x86_64-linux]
Ruby Path: /usr/bin/ruby
Install User: root

Installing slack-notifier...
Successfully installed slack-notifier-1.2.1
1 gem installed
/pipeline/pretty-slack-notify-48c80ae2-436d-4620-ab1d-b18c31b4c247/run.rb:71:in `<main>': undefined method `code' for nil:NilClass (NoMethodError)
```

**Comment by awakia** [comment]
Thank you for reporting, I'll create pull request to fix this

**Comment by dtan4** [comment]
Fixed at #25 and released as v0.2.14 :rocket: Please try this.
